opal/mca/btl/openib/help-mpi-btl-openib.txt

```diff
@@ -250,6 +252,11 @@ this run.
 
 If the "requested type" is "<any>", this usually means that *no*
 OpenFabrics devices were found within the requested bus distance.
+
+Note starting with Open MPI 4.0, only iWarp and RoCE devices are considered
+for selection by default.  Set the btl_openib_allow_ib MCA
+parameter to "true" to allow use of Infiniband devices.
+
```

Comment:
Be careful of these blank lines in help files because they get rendered to the user.